.github/workflows/24HPostsCountMoarFediverse.yaml

```diff
@@ -0,0 +1,17 @@
+name: 24HPostsCountMoarFediverse
+
+on:
+  # uncomment the following 2 lines so that it runs automatically every 15 minutes
+  # schedule:
+  #   - cron: '0 0 * * *'
+  workflow_dispatch:
+
+jobs:
+  PostsCount:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Graphana call
```

Comment:
I'd change the name to something more descriptive, something like "Retrieve number of posts indexed in the last 24 hours"

Reply:
As you wish :-)

Reply:
Are you speaking about the action "name" or the step "name" or the action filename?

Reply:
on line 13, it says "Graphana call", that's the one :)